skimage/transform/radon_transform.py

```diff
@@ -107,13 +110,14 @@ def radon(image, theta=None, circle=True):
 
     def build_rotation(theta):
         T = np.deg2rad(theta)
-        R = np.array([[np.cos(T), np.sin(T), 0],
-                      [-np.sin(T), np.cos(T), 0],
+        c, s = np.cos(T), np.sin(T)
```

Comment:
These marginal gains should be left to the PR that includes the test for preserve range.